bin/check-licenses.js

```diff
@@ -52,19 +58,38 @@ function withoutDuplicates(a) {
   return [...new Set(a)]
 }
 
+function splitColumns(s) {
```

Comment:
It would be better to explicitly refer to CSV, what do you think of `parseCSVLine`?